cves/CVE-2017-11665.yml

```diff
@@ -360,11 +364,11 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
+CWE: 20
```

Comment:
If this leads to a crash, would it be a memory-related CWE? This is a pretty broad category